tensornetwork/network_components.py

```diff
@@ -34,11 +34,11 @@ class Node:
   for a node represents the rank of that tensor.
 
   For example:
-  O : No edges means this node represents a scalar value.
-  -O : A single edge means this node is a vector.
-  -O-: Two edges represents a matrix.
-  -O- : Three edges are tensor of rank 3, etc.
-   |
+
+  * No edges means this node represents a scalar value.
+  * A single edge means this node is a vector.
+  * Two edges represents a matrix.
+  * Three edges are tensor of rank 3, etc.
```

Comment:
A node with three edges is a tensor of rank 3, etc.

(Note: There is a preference now to start using "order" instead of "rank" for the number of legs on a tensor. I don't know how much that sort of change would affect our documentation and code, but we can talk about it.)

Reply:
Yes let's discuss this later. I know we've pretty strictly been using "rank" to mean the number of axes in a tensor and "dimension" to mean the size of one of the vector spaces.